ctf/tests/lunatest.lua

```diff
@@ -0,0 +1,1142 @@
+-----------------------------------------------------------------------
```

Comment:
For now, I have a problem with using luassert alone. It doesn't recognises "describe" and "it" functions and I don't understand where they are defined. Full-text search gives me nothing.